tox.ini

```diff
@@ -196,6 +196,8 @@ markers =
     timeout: tests that need a timeout time
     localtest: local tests
     scala: scala tests
+    short_regress: short_regress tests
```

Comment:
should we rename it to be more specific? e.g., `modin_sp_short_regress` and `modin_sp_precommit`?

Reply:
Yeah that's a good idea!